src/jdk.compiler/share/classes/com/sun/source/util/DocTreeFactory.java

```diff
@@ -289,6 +290,18 @@ default InheritDocTree newInheritDocTree(ReferenceTree supertype) {
      */
     ProvidesTree newProvidesTree(ReferenceTree name, List<? extends DocTree> description);
 
+    /**
+     * Creates a new {@code RawTextTree} object, to represent a fragment of uninterpreted raw text.
+     *
+     * @param kind the kind of text
+     * @param code the code
+     * @return a {@code RawTextTree} object
+     * @throws IllegalArgumentException if the kind is not a recognized kind for raw text
```

Comment:
It took me a while to understand this comment.
For here and now, I will use `String.valueOf(kind)`
Long term, it would be good to better document `null` behavior in this API. 
[JDK-8325577](https://bugs.openjdk.org/browse/JDK-8325577)